lib/subcommands/launch.js

```diff
@@ -11,16 +12,21 @@ const commands = {};
  *                            which is going to be removed.
  * @param {number} tries [5] - The maximum number of retries before
  *                             throwing an exception.
+ * @return {string} the actual command output
  * @throws {Error} If the corresponding simctl subcommand command
  *                 returns non-zero return code.
  * @throws {Error} If the `udid` instance property is unset
  */
 commands.launchApp = async function launchApp (bundleId, tries = 5) {
+  let stdout;
   await retryInterval(tries, 1000,
-    async () => await this.exec('launch', {
-      args: [this.requireUdid('launch'), bundleId]
-    })
+    async () => {
```

Comment:
This would be clearer if done inside the retry, like
```js
return await retryInterval(tries, 1000, async () => {
  const {stdout} = await this.exec('launch', {
    args: [this.requireUdid('launch'), bundleId],
  });
  return _.trim(stdout);
});
```

Reply:
does retryInterval return the result of the looped function?

Reply:
Yes. Or throws the final exception.